tests/Integration/Command/LoadFixturesCommandTest.php

```diff
@@ -278,6 +278,16 @@ public function it_does_not_load_global_state(): void
 
     private function commandTester(array $options = []): CommandTester
     {
-        return new CommandTester((new Application(self::bootKernel($options)))->find('foundry:load-stories'));
+        // randomly choose the real command name or an alias
+        $commands = [
+            'foundry:load-fixtures',
+            'foundry:load-fixture',
+            'foundry:load-stories',
+            'foundry:load-story',
+        ];
+
+        return new CommandTester((new Application(self::bootKernel($options)))->find(
+            $commands[\array_rand($commands)]
```

Comment:
🙈 I didn't feel like a true data provider was needed here...

I get this isn't proper so I can change if desired. :)

Reply:
sometimes I like to use `#[T'estWith]` for this, but nevermind, I'm cool with this